app/javascript/src/components/Dashboard/Notes/NoteCard/CardBody.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+import { Typography } from "neetoui/v2";
+
+export default function CardBody({ note }) {
+  return (
+    <Typography style="body2" className="neeto-ui-text-gray-600 mt-1 mb-3 ">
+      {note.description}
+    </Typography>
```

Comment:
Please pass `children` as props to `CardBody`. Then, you can pass `Typography` as children from the parent component.